### PR TITLE
remove toolz.drop()

### PR DESCRIPTION
**Pull request by dholth**
### Description

Replace toolz.drop() with stdlib equivalent.

### CODE REVIEW

File: conda/activate.py [hunk 1]
```diff
@@ -203,7 +203,7 @@ def raise_invalid_command_error(actual_command=None):
             raise_invalid_command_error()
 
         command = arguments[0]
-        arguments = tuple(drop(1, arguments))
+        arguments = tuple(arguments[1:])
```
Comment (kenodegard) [inline conda/activate.py:206]:
```python
command, *arguments = arguments
```

**Review by kenodegard — CHANGES_REQUESTED** [review]
Minor suggestion but otherwise good to go

File: conda/activate.py (hunk 1 above)
Comment (dholth) [reply conda/activate.py:206]: The old code seemed really excited about keeping everything as a tuple()

Comment (kenodegard) [reply conda/activate.py:206]: yeah, I noticed, I guess leaving it as a tuple provides some level of guarantee 🤷🏼‍♂️

but in this case `arguments` is only a local var and isn't passed to another function, so I see very little potential for dangerous mutations

**Review by kenodegard — APPROVED** [review]

**Pull request merged**